### ISSUE TITLE
Permissions Denied w/ tag 11.2.3

### ISSUE THREAD

**Issue by lmgillaspie** [trion-development/docker-ng-cli-karma#13]
After the latest update (11.2.3), our Concourse job using this image and another job using trion/ng-cli-e2e started failing due to a lack of permissions. We did not make any changes to our job scripts; the only thing that changed was the image tag. For now we will downgrade to 11.2.2, but hope you can add admin privileges back to the container.

Also, when trying to run
`npm install`
we get the below error:
```
25hnpm WARN checkPermissions Missing write access to /tmp/build/7650dcdc/<app_name>/<frontend>

npm ERR! code EACCES

npm ERR! syscall access

npm ERR! path /tmp/build/7650dcdc/<app_name>/<frontend>

npm ERR! errno -13

npm ERR! Error: EACCES: permission denied, access '/tmp/build/7650dcdc/<app_name>/<frontend>'

npm ERR!  [Error: EACCES: permission denied, access '/tmp/build/7650dcdc/<app_name>/<frontend>'] {

npm ERR!   errno: -13,

npm ERR!   code: 'EACCES',

npm ERR!   syscall: 'access',

npm ERR!   path: '/tmp/build/7650dcdc/<app_name>/<frontend>'

npm ERR! }

npm ERR! 

npm ERR! The operation was rejected by your operating system.

npm ERR! It is likely you do not have the permissions to access this file as the current user

npm ERR! 

npm ERR! If you believe this might be a permissions issue, please double-check the

npm ERR! permissions of the file and its containing directories, or try running

npm ERR! the command again as root/Administrator.


npm ERR! A complete log of this run can be found in:

npm ERR!     /tmp/.npm/_logs/2021-03-04T21_24_14_154Z-debug.log
```

**Comment by everflux** [comment]
Thanks for providing feedback. I am sorry, that your build is broken.
This might be introduced by the fix for https://github.com/trion-development/docker-ng-cli/issues/21 but the change should be only in :latest not in :11.2.3
I will have a look into it.

**Comment by everflux** [comment]
@lmgillaspie could you verify that this happens with 11.2.3 and not with :latest? 
The 11.2.3 image still runs as uid 0 if not started otherwise (docker -u, kubernetes securityContext.runAsUser).

**Comment by lmgillaspie** [comment]
Hi! Thank you for the super quick reply and fix! That was totally my bad for assuming 11.2.3 was latest. To your point, our job now works with the fix you made for :latest. Thank you for your help!
